platform/x11/joypad_linux.cpp

```diff
@@ -444,10 +444,10 @@ InputDefault::JoyAxis JoypadLinux::axis_correct(const input_absinfo *p_abs, int
 		jx.min = -1;
 		if (p_value < 0) {
 			jx.value = (float)-p_value / min;
+		} else {
```

Comment:
This will change the logic as previously the `p_value < 0` branch was always overridden. So we should test to make sure that the change is actually a bugfix.

Reply:
CC @Hinsbart

Reply:
Reading the code I would *assume* that the new logic is correct, as if `p_value` is negative it will return a negative number normalized by `min`, and if it's positive it will return a positive number normalized by `max`, which I would say is probably expected.

I'll try to test with my joypad to confirm.